odoo/addons/test_new_api/models/test_new_api.py

```diff
@@ -1399,3 +1399,34 @@ class TriggerRight(models.Model):
     def _compute_left_size(self):
         for record in self:
             record.left_size = len(record.left_ids)
+
+class Crew(models.Model):
+    _name = 'test_new_api.crew'
+    _description = 'All yaaaaaarrrrr by ship'
+    _table = 'test_new_api_crew'
+
+    pirate_id = fields.Many2one('test_new_api.pirate')
+    prisonner_id = fields.Many2one('test_new_api.prisonner')
+    ship_id = fields.Many2one('test_new_api.ship')
```

Comment:
This looks like failed modeling to me, like the disjoint union of a pirate-ship relation and a prisoner-ship relation.  It is at least not correct _relational_ modeling, in the strict sense.  Is this really what you want to support, or simply a misleading impression on the actual model?